<?xml version="1.0" encoding="UTF-8"?>
<!--
  Copyright 2010, 2011 Christian Surlykke

  This file is part of ProxyManager.

  PitchTrainer is free software: you can redistribute it and/or modify
  it under the terms of the GNU General Public License as published by
  the Free Software Foundation, version 3

  PitchTrainer is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  GNU General Public License for more details.

  You should have received a copy of the GNU General Public License
  along with PitchTrainer.  If not, see <http://www.gnu.org/licenses/>.
-->
<ui version="4.0">
 <class>ProfileManager</class>
 <widget class="QDialog" name="ProfileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Manage Profiles&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListView" name="profileList">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>40</y>
     <width>177</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="profileBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>177</y>
     <width>377</width>
     <height>411</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="useProxyCheckBox">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>61</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Use a proxy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="profileNameInput">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>25</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>98</y>
      <width>39</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>http://</string>
    </property>
   </widget>
   <widget class="QFrame" name="proxyData">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>89</y>
      <width>360</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="hostExceptionsList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>256</width>
       <height>101</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>37</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>No proxy for these hosts</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newHostExceptionButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteHostExceptionButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newDomainExceptionButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>200</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteDomainExceptionButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>230</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QListWidget" name="domainExceptionsList">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>199</y>
       <width>256</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>182</y>
       <width>270</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>No proxy for these domains</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>212</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="proxyHostInput"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="proxyPortInput">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>proxyData</zorder>
   <zorder>useProxyCheckBox</zorder>
   <zorder>label_3</zorder>
   <zorder>profileNameInput</zorder>
  </widget>
  <widget class="QDialogButtonBox" name="profileManagerButtonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>610</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="newProfileButton">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>140</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteProfileButton">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>140</y>
     <width>86</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>130</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="client.qrc">:/icons/icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>profileList</tabstop>
  <tabstop>newProfileButton</tabstop>
  <tabstop>deleteProfileButton</tabstop>
  <tabstop>profileNameInput</tabstop>
  <tabstop>useProxyCheckBox</tabstop>
  <tabstop>proxyHostInput</tabstop>
  <tabstop>proxyPortInput</tabstop>
  <tabstop>newHostExceptionButton</tabstop>
  <tabstop>deleteHostExceptionButton</tabstop>
  <tabstop>newDomainExceptionButton</tabstop>
  <tabstop>deleteDomainExceptionButton</tabstop>
  <tabstop>domainExceptionsList</tabstop>
  <tabstop>hostExceptionsList</tabstop>
  <tabstop>profileManagerButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="client.qrc"/>
 </resources>
 <connections/>
</ui>
